addons/web/static/src/core/utils/hooks.js

```diff
@@ -46,7 +48,7 @@ export function useAutofocus({ refName, selectAll } = {}) {
     // LEGACY
     useEffect(
         (el) => {
-            if (el) {
+            if (el && (!uiService.activeElement || uiService.activeElement.contains(el))) {
```

Comment:
@bastien-ipb 
Just curious here, which case did you encounter where the uiService.activeElement was undefined ?
cc @fdardenne

Reply:
I think this is never the case, so we could have simply written
```suggestion
            if (uiService.activeElement.contains(el)) {

Reply:
In the context of some QUnit, like this one: [swipe settings on larger screen sizes has no effect [REQUIRE TOUCHEVENT] ](https://github.com/odoo/odoo/blob/d5c3d47f80d88301e2311c9c1d6eabedca8a562d/addons/web/static/tests/mobile/webclient/settings_form_view_tests.js#L107)